releasenotes/notes/add-dm-trace-tag-471f5ee0303a8497.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add sampling mechanism trace tag
```

```suggestion
    internal: Add sampling mechanism trace tag
```

? I am actually not sure here, just curious if someone reading this will understand what it means/what they get from it.